File: UsuariosApp.Domain/Dtos/Responses/AutenticarUsuarioResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UsuariosApp.Domain.Dtos.Responses
{
    public record AutenticarUsuarioResponse(
            Guid Id,
            string Nome,
            string Email,
            DateTime DataHoraAcesso,
            DateTime DataHoraExpiracao,
            string AccessToken,
            string Perfil
        );
}

File: src/main/java/br/com/cotiinformatica/dtos/responses/AutenticarUsuarioResponse.java
package br.com.cotiinformatica.dtos.responses;

import java.time.LocalDateTime;
import java.util.UUID;

public record AutenticarUsuarioResponse(
        UUID id, //Id do usuario autenticado
        String nome, //Nome do usuario
        String email, //Email do usuario
        String perfil, //Perfil de acesso do usuario
        LocalDateTime dataHoraAcesso, //Data e hora do acesso
        String acessToken //TOKEN JWT (credencial)
) {
}
